Sources/FingerprintJS/Harvesters/LocalAuthenticationInfoHarvester/Providers/AuthenticationContextInfoProviding.swift

```diff
@@ -0,0 +1,31 @@
+#if canImport(LocalAuthentication)
+import LocalAuthentication
+#endif
+
+@available(tvOS, unavailable)
+protocol AuthenticationContextInfoProviding {
+    var isPasscodeEnabled: Bool { get }
+    var isBiometricsEnabled: Bool { get }
+    var supportedBiometryType: BiometryType { get }
+}
+
+#if os(iOS)
+extension LAContext: AuthenticationContextInfoProviding {
+
+    var isPasscodeEnabled: Bool {
+        canEvaluatePolicy(.deviceOwnerAuthentication, error: nil)
+    }
+
+    var isBiometricsEnabled: Bool {
+        canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: nil)
+    }
+
+    var supportedBiometryType: BiometryType {
+        if biometryType == .none {
+            _ = canEvaluatePolicy(.deviceOwnerAuthentication, error: nil)
+        }
```

Comment:
This seems like a no-op. Maybe a missing early return?

Reply:
This one is intentional. It is there to ensure the `biometryType` property has the correct value, as it is only set after we  call the `canEvaluatePolicy(_:error:)` method. See [the docs](https://developer.apple.com/documentation/localauthentication/lacontext/2867583-biometrytype) for more details. 

This bahavior of `LAContext` is rather unclear, so I decided to add a code comment explaining the reasoning behind the above code.